src/cursor/cur_std.c

```diff
@@ -567,6 +579,36 @@ __wt_cursor_get_valuev(WT_CURSOR *cursor, va_list ap)
     API_END_RET_STAT(session, ret, cursor_get_value);
 }
 
+/*
+ * __wt_cursor_get_raw_key_value --
+ *     WT_CURSOR->get_raw_key_value default implementation
+ *
+ */
+int
+__wt_cursor_get_raw_key_value(WT_CURSOR *cursor, WT_ITEM *k, WT_ITEM *v)
+{
+    WT_DECL_RET;
+    WT_SESSION_IMPL *session;
+
+    CURSOR_API_CALL(cursor, session, get_value, NULL);
+
+    if (!F_ISSET(cursor, WT_CURSTD_VALUE_EXT | WT_CURSTD_VALUE_INT))
+        WT_ERR(__wt_cursor_kv_not_set(cursor, false));
+
+    /* Force an allocated copy when using cursor copy debug. */
+    if (FLD_ISSET(S2C(session)->debug_flags, WT_CONN_DEBUG_CURSOR_COPY))
+        WT_ERR(__wt_buf_grow(session, &cursor->value, cursor->value.size));
+
+    k->data = cursor->key.data;
```

Comment:
Is it worth supporting a semantic here where a user can pass a NULL for either key or value to retrieve just one or the other?

Reply:
That sounds like a useful idea - I'll make the change.

Reply:
Done!